cli/tests/unit/serve_test.ts

```diff
@@ -693,24 +693,30 @@ function createStreamTest(count: number, delay: number, action: string) {
       onError: createOnErrorCb(ac),
     });
 
-    await listeningPromise;
-    const resp = await fetch(`http://127.0.0.1:${servePort}/`);
-    const text = await resp.text();
+    try {
```

Comment:
HTTP behaves more like you would expect now -- it will always return EOF if there is an error in the middle of a stream (matching the behaviour if there was an error in any other resource).